src/index.js

```diff
@@ -7,9 +7,14 @@ import errorhandler from 'errorhandler';
 import router from './routes/index';
 import ErrorHandler from './utils/error';
 import logger from './utils/winston';
+import config from './config';
 
 const isDevelopment = process.env.NODE_ENV;
```

Comment:
This line calls '.env' directly. Maybe you can use the config object you've already imported.

Reply:
Oh yes.Corrected now